src/ontodia/data/sparql/sparqlDataProviderSettings.ts

```diff
@@ -231,15 +265,15 @@ export const OWLStatsSettings : SparqlDataProviderSettings = {...OWLRDFSSettings
         `,
 }};
 
-export const DBPediaSettings : SparqlDataProviderSettings = {...OWLRDFSSettings, ... {
+export const DBPediaSettings: SparqlDataProviderSettings = {...OWLRDFSSettings, ... {
     ftsSettings: {
```

Comment:
Should be `fullTextSearch` here?
Use `{...A, b: 1, c: 1}` instead of `{...A, ...{ b: 1, c: 2 }}` to help compiler catch typos.